CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - Faster state queries by avoiding locking the block state file when reading.
 - Fix a bug by shutting down RPC before the node, which caused the node to crash
   when attempting a graceful shutdown while processing RPC requests.
+- The node now drops all connections on an unrecognized protocol update and
+  refuses to accept new transactions.
 
```

Comment:
Missing a log for forcing down the rpc after 10 seconds.

Reply:
I don't think this change necessarily should be in the changelog. I wouldn't expect the rpc to be open after the node has shutdown. Also there is the other entry already. "Fix a bug by shutting down the RPC...."

Reply:
I agree since we did not release the version which had this "never terminate due to idle connections" bug. There is no reason to have two entries in the changelog for this.